concepts/functions/about.md

```diff
@@ -170,49 +198,56 @@ Now, we can call the functions in the order we want.
 
 ## Scope of Variables
 
-If variable is defined inside a function, then it will be only accessible inside the function. If we want to access the variable outside the function, we need to use the [`global`][global] keyword. [`nonlocal`][nonlocal] keyword is used to access the variable inside a nested function.
+If variable is defined inside a function, then it will be only accessible inside the function.
+If we want to access the variable outside the function, we need to use the [`global`][global] keyword.
```

Comment:
```suggestion
If we want to access the variable name outside the function scope (_or namespace_), we need to use the [`global`][global] keyword.
```

Reply:
_ooof_ not sure here if we should go into the dangers of this or just link to them.  The `global` statement is pretty complicated: https://docs.python.org/3/reference/simple_stmts.html#the-global-statement.  What is especially concerning is that it is read in before the run starts and lasts for the life of the program.  The namespace collision risk is .... ugly.

Reply:
Yeah, I don't do a happy dance re global variables. Some kind of warning is probably a good thing to implant...